project/cmake/addons/CMakeLists.txt

```diff
@@ -228,10 +228,22 @@ endif()
 # error either in ADDONS_TO_BUILD or in the directory configuration.
 set(SUPPORTED_ADDON_FOUND FALSE)
 
+message(STATUS "BUILD_ARGS=${BUILD_ARGS}")
```

Comment:
Wasn't there also a separate PR for this commit? please remove those debug messages. they are not useful for the user of the module.